src/Codeception/Lib/DbPopulator.php

```diff
@@ -98,8 +120,35 @@ public function run()
         return true;
     }
 
+    private function buildCommands()
+    {
+        if ($this->commands !== null) {
+            return $this->commands;
+        } else if (!isset($this->config['dump'])) {
+            return $this->buildCommand($this->config['populator']);
```

Comment:
I think that this is a bug, `run` method expects to get array from `buildCommands` but it returns string.
Wrapping it to array, would allow to simpify `run` by removing the first condition.